<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZEA_Lab2</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_A">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_B">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_X">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>task.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Run">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Решить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Answer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Y = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_B">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_X">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clean">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчистить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
